packages/web/src/providers/snackbar-provider/snackbar/contents/ReceiveWugnotContent.tsx

```
@@ -0,0 +1,41 @@
+  };
+
+  return (
+    <div className="notice-body clickable" onClick={onClick}>
+      <div className="icon-wrap-wrapper">
+        <IconWrap className="icon-wrap" />
+      </div>
+      <div>
+        <h5>{content?.title ? content.title : t("Modal:toast.receive-wugnot.title")}</h5>
+        <div
+          className="description"
+          dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)